Recent Fedora releases have disabled export ciphers by default, so
don't test that they work.

diff -ur M2Crypto/tests/test_ssl.py M2Crypto-0.21.1/tests/test_ssl.py
--- M2Crypto/tests/test_ssl.py	2014-01-06 22:35:45.777935677 +0100
+++ M2Crypto-0.21.1/tests/test_ssl.py	2014-01-06 22:43:34.025594902 +0100
@@ -463,6 +463,7 @@
         finally:
             self.stop_server(pid)
         
+    @unittest.skip("Export ciphers are prohibited in recent Fedora releases")
     def test_use_weak_cipher(self):
         if fips_mode: # Weak ciphers are prohibited
             return
